docs/release-notes/self-hosted/self-hosted-v7.0.0.md

```diff
@@ -64,6 +64,12 @@ However, until now the Codacy permissions for this role were limited to the same
 
 -   Improved the loading times of the Organization Overview and Repositories list pages using a short-lived cache in the user's browser. (CY-5793)
 
+-   Now, if your organization has **less than 100 repositories** it's possible to sort the [**Repositories list** page](https://docs.codacy.com/v7.0/organizations/managing-repositories/) using the available columns. (CY-5695)
+
+-   It's now possible to click on the organization overview chart to drill down on the information and understand which repositories require attention to improve the organization's code quality. (CY-5580)
```

Comment:
I'd avoid using two bullets basically starting with the same wording "Now", "it's now"... And try to reword it.

On the first I'd also focus on the feature first and only then say the limitation (only when you have less than 100 repos), creates the felling of something nice instead of being immediately confronted with a technical limitation.

Reply:
Nice suggestions @vascorsd, I have just applied your feedback. :+1: